blocks/search.py

```diff
@@ -263,10 +263,19 @@ def search(self, input_values, eol_symbol, max_length,
             first iteration are ignored. This useful when the sequence
             generator was trained on data with identical symbols for
             sequence start and sequence end.
+        as_lists : bool, optional
+            If ``True``, a list of trimmed output sequences and a list of
+            their costs are returned, instead of the default output format.
 
         Returns
         -------
-        Sequences in the beam, masks, and corresponding log-probabilities.
+        outputs : :class:`numpy.ndarray`
+            The generated sequences, time is the first axis, the number in
+            the beam is the second.
+        mask : :class:`numpy.ndarray`
+            The mask for the outputs, same layout as `outputs`
```

Comment:
I think it is a good idea to repeat here, that the `mask` is returned only if `as_lists=False`.